multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -3147,3 +3155,28 @@ def __init__(self, **kwargs):
             "raise an exception starting in v0.7."
         )
         super(AutoMMPredictor, self).__init__(**kwargs)
+
+
+class MultiModalOnnxPredictor(MultiModalPredictor):
+    def __init__(self, model):
+        import onnxruntime as ort
+
+        self.sess = ort.InferenceSession(model.SerializeToString(), providers=["CPUExecutionProvider"])
+
+    def predict(self, data: Union[pd.DataFrame, dict, list, str]):
+        raise NotImplementedError()
```

Comment:
Wondering why we left this unimplemented?

Reply:
This depends on problem_type / postprocessing, which isn't available within the converted onnx model.